CHANGES

```diff
@@ -1,3 +1,15 @@
+Release 1.3 (in development)
+----------------------------------
+
+.. rubric:: Enhancements
+
+* Add bundle streaming support [#130]
```

Comment:
Maybe a bit more verbose (we have not so many changes anyway :smirk:)?

```suggestion
* Add option ``enable_streaming=true`` to allow using RAUC's HTTP(S) bundle streaming capabilities instead of downloading and storing bundles separately. [#130]
```

Reply:
Will do.